packages/core/src/external/athenahealth/index.ts

```diff
@@ -51,18 +60,49 @@ function getS3UtilsInstance(): S3Utils {
   return new S3Utils(region);
 }
 
+const problemStatusesAthena = ["CHRONIC", "ACUTE"];
+// TODO import from shared?
+const vitalSignCodesMapAthena = new Map<string, string>();
+vitalSignCodesMapAthena.set("8310-5", "VITALS.TEMPERATURE");
+vitalSignCodesMapAthena.set("8867-4", "VITALS.HEARTRATE");
+vitalSignCodesMapAthena.set("9279-1", "VITALS.RESPIRATIONRATE");
+vitalSignCodesMapAthena.set("2708-6", "VITALS.INHALEDO2CONCENTRATION");
+vitalSignCodesMapAthena.set("8462-4", "VITALS.BLOODPRESSURE.DIASTOLIC");
+vitalSignCodesMapAthena.set("8480-6", "VITALS.BLOODPRESSURE.SYSTOLIC");
+vitalSignCodesMapAthena.set("29463-7", "VITALS.WEIGHT");
+vitalSignCodesMapAthena.set("8302-2", "VITALS.HEIGHT");
+vitalSignCodesMapAthena.set("39156-5", "VITALS.BMI");
+
 export type AthenaEnv = "api" | "api.preview";
 export function isAthenaEnv(env: string): env is AthenaEnv {
   return env === "api" || env === "api.preview";
 }
 
+// TYPES FROM DASHBOARD
```

Comment:
What does it mean? The BE should not rely on types from the dash, but the other way around, IMO.

Reply:
Will leave as is for now.